models/CreateNewPatientRequest.js

```diff
@@ -4,18 +4,19 @@ Patient {
 
   family : string
   given : string
-  suffix : string
-  phone : {
+  --middle : string
+  --suffix : string
+  phone : [{
     value : string
     use : enum (home, work, temp, old, mobile)
-  }
+  }]
   email : array of strings
   gender : enum (male, female, other, unknown)
   birthdate : string
   race : enum (American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Black or African-American, White, Other Race)
   ethnicity: enum (Hispanic, Not Hispanic)
```

Comment:
Do we need to explicitly include the 3rd option, Unknown?  Or perhaps that is implied if this field is null/blank

Reply:
The enum for gender is the required valueset, but it doesnt mean we have to use the unknown value. If gender is optional for registration, we would use this value if gender is not specified. If gender is required and we allow users to decline to answer, I guess unknown would be used in this situation as well.